library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -1215,11 +1215,15 @@ public void onMapReady(MapboxMap mapboxMap) {
 
     @Override
     public boolean changeLocationUpdates(long updateInterval, long fastestUpdateInterval, int accuracyLevel) {
-        if (updateInterval > -1 && fastestUpdateInterval > -1 && getLocationClient() != null) {
+        if (updateInterval > -1 && fastestUpdateInterval > -1 && (accuracyLevel == LocationRequest.PRIORITY_HIGH_ACCURACY
```

Comment:
If the user will never have any option other than `LocationRequest.PRIORITY_HIGH_ACCURACY`, then there is no need to have an accuracyLevel argument.

Keeping it as it is would mean that the user will be unable to set the other parameters if they mistakenly set a lower priority. 

Is this desirable?